certbot/main.py

```diff
@@ -653,6 +653,10 @@ def main(cli_args=sys.argv[1:]):
     config = configuration.NamespaceConfig(args)
     zope.component.provideUtility(config)
 
+    # -v should imply --text
+    if config.verbose_count > cli.flag_default("verbose_count"):
+        config.text_mode = True
```

Comment:
This should be moved to `HelpfulArgumentParser.parse_args` and should not be set if `dialog_mode` is set.